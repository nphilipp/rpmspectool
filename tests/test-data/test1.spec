# Just a run-of-the-mill spec file using some macros and corner cases
%bcond_without subpackage
%bcond_with dependency

Name: test
Version: 1
Release: 1
Summary: Test spec file
Group: Old RPM versions require group, so we test it

License: CC-0
URL: https://github.com/some/example/repo
Source0: %{url}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
Patch: %{url}/commit/0123456789abcdef0123456789abcdef01234567.patch
Patch3: boop.patch
Patch: %{url}/commit/abcdef0123456789abcdef0123456789abcdef01.patch
%if %{with dependency}
BuildRequires: some-dependency
%endif
BuildArch: noarch

%if %{with subpackage}
%package subpackage
%endif

%description
%{summary}.

%if %{with subpackage}
%description subpackage
%{summary} subpackage.
%endif

%prep
%autosetup

%build
%make_build

%install
%make_install

%files
%license LICENSE
%doc AUTHORS README.md

%changelog
