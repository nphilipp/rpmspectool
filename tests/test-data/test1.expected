Source0: https://github.com/some/example/repo/archive/v1.tar.gz#/test-1.tar.gz
Patch0: https://github.com/some/example/repo/commit/0123456789abcdef0123456789abcdef01234567.patch
Patch3: boop.patch
Patch4: https://github.com/some/example/repo/commit/abcdef0123456789abcdef0123456789abcdef01.patch
